### ISSUE TITLE
The option 'android.enableUnitTestBinaryResources' is deprecated

### ISSUE THREAD

**Issue by abarke** [aws-amplify/amplify-flutter#2962]
### Description

$ flutter build apk

💪 Building with sound null safety 💪

FAILURE: Build completed with 2 failures.

1: Task failed with an exception.
-----------
* Where:
Build file 'C:\Users\Dude\AppData\Local\Pub\Cache\hosted\pub.dev\amplify_secure_storage-0.3.0+3\android\build.gradle' line: 24

* What went wrong:
A problem occurred evaluating project ':amplify_secure_storage'.
> Failed to apply plugin 'com.android.internal.library'.
   > The option 'android.enableUnitTestBinaryResources' is deprecated.
     The current default is 'false'.
     It has been removed from the current version of the Android Gradle plugin.
     The raw resource for unit test functionality is removed.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
==============================================================================

2: Task failed with an exception.
-----------
* Where:
Script 'C:\Users\Dude\Code\flutter\packages\flutter_tools\gradle\flutter.gradle' line: 407

* What went wrong:
A problem occurred configuring project ':amplify_secure_storage'.
> Failed to notify project evaluation listener.
   > Could not get unknown property 'android' for project ':amplify_secure_storage' of type org.gradle.api.Project.
   > Could not get unknown property 'android' for project ':amplify_secure_storage' of type org.gradle.api.Project.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
==============================================================================

* Get more help at https://help.gradle.org

BUILD FAILED in 6m 1s
Running Gradle task 'assembleRelease'...                          363,3s
Gradle task assembleRelease failed with exit code 1

### Categories

- [ ] Analytics
- [ ] API (REST)
- [ ] API (GraphQL)
- [ ] Auth
- [ ] Authenticator
- [ ] DataStore
- [X] Storage

### Steps to Reproduce

1. `amplify init`
2. `amplify add auth`

Various files are changed, including `.flutter-plugins`:

```
amplify_secure_storage=C:\\Users\\Dude\\AppData\\Local\\Pub\\Cache\\hosted\\pub.dev\\amplify_secure_storage-0.3.0+3\\
```

### Solution

The solution is to navigate to the following file: `C:\Users\Dude\AppData\Local\Pub\Cache\hosted\pub.dev\amplify_secure_storage-0.3.0+3\android\gradle.properties` and remove the only line `android.enableUnitTestBinaryResources=true` which is silly, as one would have to do this after every `flutter  pub get`

### Platforms

- [ ] iOS
- [X] Android
- [ ] Web
- [ ] macOS
- [ ] Windows
- [ ] Linux

### Flutter Version

3.7.10

### Amplify Flutter Version

amplify_flutter-1.0.0

### Deployment Method

Amplify CLI

**Comment by Jordan-Nelson** [comment]
You can find the version info in About Android Studio.

<img width="595" alt="Screen Shot 2023-05-02 at 1 13 05 PM" src="https://user-images.githubusercontent.com/20613561/235739059-ed3c7e17-7327-4b73-b9e2-00d2da6e2968.png">

**Comment by Jordan-Nelson** [comment]
@abarke - I have been unable to reproduce this with the latest stable versions of flutter & Android Studio. Can you share the following info:
- The version of Android Studio you are using
- The version of flutter you are using
- The contents of `android/build.gradle`

**Comment by Jordan-Nelson** [comment]
@abarke - Version 1.1.0 of the amplify libraries includes a fix for this. v1.1.0 requires flutter 3.10. If you need a fix in 1.0.x, let me know. I am going to close this out.
